sidekick_core/lib/src/template/sidekick_package.template.dart

```diff
@@ -81,12 +78,6 @@ class SidekickTemplateProperties {
   /// When there's a flutter package that requires a flutter sdk
   final bool? shouldSetFlutterSdkPath;
 
-  /// When the dart package is located in root of the repo
-  final bool? isMainProjectRoot;
-
-  /// true when a /packages directory exists
-  final bool? hasNestedPackagesPath;
```

Comment:
removal of fields is, unfortunately, a breaking API change. We can't just remove them, it would break existing `sidekick` clis.